composer.json

```diff
@@ -29,7 +29,7 @@
         "ext-json": "*",
         "ext-dom": "*",
         "ext-simplexml": "*",
-        "phpunit/phpunit": "~4.0|~5.0",
+        "phpunit/phpunit": "~4.8",
```

Comment:
4.8 is pretty far out of date at this point and isn't guaranteed to be compatible with PHP 7 or 7.1. To make sure the tests are still compatible with all of the versions we test in Travis, I would recommend `^4.8.35|^5.4.0|^6.0.0`